Observation/Observation-Cellularity-Example.json

```diff
@@ -18,16 +18,14 @@
       "value": "RA257630"
     }
   },
- "interpretation": [
-    {
-      "coding": [
-        {
-          "system": "http://terminology.hl7.org/CodeSystem/v3-ObservationInterpretation",
-          "code": "H",
-          "display": "High"
-        }
-      ]
-    }
-  ],
+  "valueCodeableConcept": {
+    "coding": [
+      {
+        "system": "http://terminology.hl7.org/CodeSystem/v3-ObservationInterpretation",
```

Comment:
Codes should use the SNOMED code you've chosen in the concept map. Other than that, the PR is good